web/app/js/components/LineGraph.jsx

```diff
@@ -133,6 +135,40 @@ export default class LineGraph extends React.Component {
     }
   }
 
+  flashLatestDataPoint() {
+    this.svg.select("circle").remove();
+    let circleData = {};
+
+    if (!_.isEmpty(this.props.data) && this.props.showFlash) {
+      circleData = this.props.data.slice(-1)[0];
+      this.svg.append("circle")
+        .attr("class", "flash")
+        .style("opacity", 0.6)
+        .attr("cx", () => {
+          if (circleData.timestamp) {
+            return this.xScale(circleData.timestamp);
+          }
+        })
+        .attr("cy", () => {
+          if (circleData.value) {
+            return this.yScale(circleData.value);
+          }
+        })
+        .attr("r", 6)
+        .transition()
+        .on("start", function repeat() {
+          d3.active(this)
+            .transition()
+            .duration(550)
+            .style("opacity", 0.6)
+            .transition()
+            .duration(550)
```

Comment:
I think a duration of ~3000 is a little calmer :)

Reply:
I'll try that once I've made siggys suggested change (```Perhaps there's a way to not add/remove the circle on updateGraph(), but rather just shift its position? This way the flashing will be seamless across data updates.```), since otherwise it will never fade out on the servicemesh page, as that one refreshes every 2secs.